source/JsonObject.cpp

```diff
@@ -16,26 +16,24 @@ namespace Aws
     {
         JsonObject::JsonObject() : m_wasParseSuccessful(true) { m_value = nullptr; }
 
-        JsonObject::JsonObject(cJSON *value)
-            : m_value(cJSON_Duplicate(value, 1 /* recurse */)), m_wasParseSuccessful(true)
+        JsonObject::JsonObject(aws_json_value *value)
+            : m_value(aws_json_value_duplicate(value)), m_wasParseSuccessful(true)
         {
         }
 
         JsonObject::JsonObject(const String &value) : m_wasParseSuccessful(true)
         {
-            const char *return_parse_end;
-            m_value = cJSON_ParseWithLengthOpts(value.c_str(), value.length(), &return_parse_end, 0);
+            m_value = aws_json_value_new_from_string(ApiAllocator(), aws_byte_cursor_from_c_str(value.c_str()));
```

Comment:
```suggestion
            m_value = aws_json_value_new_from_string(ApiAllocator(), ByteCursorFromString(value));
```

String knows how long it is, use this helper function to avoid the O(N) strlen() under the hood